gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -62,17 +64,85 @@
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
   private static final String GCD = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
The only thing I was suggesting is to re-use constants as in this case 2 are having the `"v1beta2"` string.
These are internal constants, and I think It would make it more readable and maintainable.
Something like:
```java
private static final String GCD_VERSION = "v1beta2";
private static final String GCD_BUILD = "rev1-2.1.2b";
private static final String GCD = "gcd-" + GCD_VERSION + "-" + GCD_BUILD;
private static final String GCD_FILENAME = GCD + ".zip";

```